tutorial/backoffice/add-zone-structure.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Add Zone Structure
+description: This document outlines creating zone structure in Backoffice
+sidebar_position: 1
+---
+
+An important step in setting up PlaceOS is to create your organizations physical structure as [Zones](../../overview/zones.md).
+
+You can think of Zones as creating your digital twin of your physical organization structure.
+
+The three most common zones required are:
+* `org` zone: your parent zone, named after your organization
+* `building` zone: your buildings within your organization
+* `level` zone: the levels (floors) within your buildings
+
+In the following steps, we will walkthrough configuring your Zone structure including Organization, Building and Levels. 
+
+## Prerequisites
+
+* PlaceOS Backoffice Administrator Access
+* [SVG Floor Plans](../../how-to/user-interfaces/svg-map-creation.md) (optional)
+
+## Org Zone
+
+An Org Zone is the parent zone for all other zones.
+
+The Org Zone is typically named after the organization.
+
+### Steps to create the org zone
+
+1. Login to PlaceOS Backoffice
+2. Navigate to Zones
+3. Click the `+` button to create a new zone  
+![Add new org zone](./assets/new-zone-button.png)  
+4. In the New Zone Modal, you will need to enter the following information:
+- **Parent Zone**: Your new org zone will be the parent, leave this blank
+- **Name**: The name of your org zone (typically your organization name)
+- **Display Name**: How you want the zone to appear to end users (optional)
+- **Tags**: Enter a single tag of `org`
+
+The remaining fields are not required for org zones:
+- **Description**: Optional zone description
+- **Location**: GPS Coordinates for the building
+- **Code**: Cost Center or Building Code
+- **Type**: Code category
+- **Capacity**: Total building capacity
```

Comment:
maybe change `Total building capacity` to `primarily for defining level capacity`

Reply:
fixed in c8a1dfd